### ISSUE TITLE
Totenwerfer Buff

### ISSUE THREAD

**Issue by RhynxSyn** [Reagy/TF2Classic-KO-Custom-Weapons#132]
Since the Totenwerfer is supposed to be similar to the Quick-Fix, it'd be understandable for it to have +10% ubercharge rate.

Though in stock TF2 the Quick-Fix TECHINCALLY has a +60% ubercharge rate. And honestly, this is the only reason why I use the Quick-Fix at all. If it had only +10%, it wouldn't be used that much. Now while the geared up ghost exists, that isn't really a good reason. As even though that the Totenwerfer has the -50% max overheal, it doesn't give the bonus ubercharge rate for having below the certain overheal limit for some reason.

**_TL;DR:_**
The totenwerfer has a +10% ubercharge rate while the Quick-Fix has a +60% ubercharge rate due to the smaller overheal, it'd be good if the totenwerfer got +60% instead of +10%.

**Comment by Theoneofwonders** [comment]
This makes no sense.

How can the Quick-Fix have a "60% faster ubercharge rate" if it's actually 10%?

Clarify.

**Comment by Theoneofwonders** [comment]
I will test this in 2.1 to see how fast it takes the Totenwerfer and Stock to gain Ubercharge but it should work like it does in live.

**Comment by Theoneofwonders** [comment]
Tested with the upcoming hotfix, it provide the expected 60% uber rate.

Closing this issue.
